react-frontend/src/Home.js

```diff
@@ -1,16 +1,36 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import ButtonAppBar from './Navbar';
 
-const HomePage = () => (
-  <>
-    <ButtonAppBar />
-    <header>
-      <h1>My Simple React Home Page</h1>
-    </header>
-    <main>
-      <h1>Mission Log</h1>
-    </main>
-  </>
-);
+const HomePage = () => {
+  const [text, setText] = useState('');
+  const fullText = 'Welcome to Mission Log';
+
+  useEffect(() => {
+    if (text.length < fullText.length) {
+      setTimeout(() => setText(fullText.slice(0, text.length + 1)), 100);
+    }
+  }, [text]);
+
+  return (
+    <div style={{
```

Comment:
Though inline styling is good for styling components that have very unique styles it may end up becoming repetitive if other pages may require a similar styling. I usually define my styles in another file such as App.css and import the styles into the react file. Also look into css classes and id's they allow you to define styles for similar components.

Reply:
Fixed!